terraform/aws/scripts/install.sh.tpl

```diff
@@ -3,22 +3,20 @@ set -e
 
 sudo apt-get update -y
 sudo apt-get install -y curl unzip
-curl -L "${download-url}" > /tmp/approzium.zip
+curl -o approzium.zip -LO "${download-url}"
 
-cd /tmp
```

Comment:
I stopped using the `/tmp` directory in the script because I got permissions errors when I tried it by hand, so instead I switched to using the default home directory instead where there are no permissions errors.